elasticsearch/dictionaries/synonyms/_all.txt

```diff
@@ -0,0 +1,3 @@
+# brands
+mozilla, modzilla, mozzila, mozzilla, mozila
+ios, ipad, iphone, ipod
```

Comment:
is `ipod` needed here?

Reply:
discussed in meeting: you can run firefox on an ipod touch!